feeds/combiner.go

```diff
@@ -235,12 +254,14 @@ func (cf *combinerFeed) fetchAlertsAndHandle(
 			continue
 		}
 
-		if _, exists := cf.alertCache.Get(alert.Alert.Hash); exists {
```

Comment:
[fix] if 2 bots subscribe to same bot, first to query will be dispatched but the next ones will be ignored because they are cached. Added subscriber bot id to hash key